plugin/loader/preload.go

```diff
@@ -20,4 +21,5 @@ func init() {
 	Preload(pluginflatfs.Plugins...)
 	Preload(pluginlevelds.Plugins...)
 	Preload(pluginpeerlog.Plugins...)
+	Preload(pluginfxtest.Plugins...)
```

Comment:
It's a bit awkward that we have useless test code compiled into the binary hidden with code hidden behind an env var, although not the end of the world if it's just way easier than the alternative.

Reply:
I completely agree but I can't think of a feasible alternative that won't be a huge time suck. This feels yucky but AFAIU in practice it has insignificant impact.